addons/vagrant/helper_scripts/config_switch.sh

```diff
@@ -65,6 +65,10 @@ auto swp15
 iface swp15
     bridge-access 18
 
+auto swp16
+iface swp16
+    bridge-access 17
+
```

Comment:
Nothing for swp17? (is it due to config from addons/vagrant/helper_scripts/config_node03.sh)

Reply:
Nothing, we configure it during test suite (like swp12).